lib/segment/src/data_types/named_vectors.rs

```diff
@@ -146,7 +146,10 @@ impl<'a> NamedVectors<'a> {
                     let preprocessed_vector = distance.preprocess_vector(v.to_vec());
                     *vector = CowValue::Dense(Cow::Owned(preprocessed_vector))
                 }
-                CowValue::Sparse(_) => {}
+                CowValue::Sparse(v) => {
```

Comment:
I find this place of sorting very untrivial. Can we do sort in `SparseVectors::new` before validation?

Reply:
And do sorting while serde deserialization

Reply:
There is indeed a problem with the current implementation.
It only makes sure that the vector are sorted on insertion and updates.

I still need to add a sorting instruction for the query vectors when they reach the `SparseRawScorer` :hammer: